alarm/utils.py

```diff
@@ -33,3 +33,55 @@ def any_of_trade_pair_thresholds_is_broken(trade_pair):
             return True
 
     return False
+
+
+def create_thresholds_brakes_from_recent_candles_update(trade_pair):
+    thresholds = Threshold.objects.filter(trade_pair=trade_pair)
+    for threshold in thresholds:
+        ThresholdBrake.objects.create(threshold=threshold)
+
+
+def create_message_for_threshold_break(trade_pair):
+    three_second_ago = timezone.now() - timedelta(seconds=3)
+
+    messages = {}
+
+    trade_pair_str = format_trade_pair_for_message(trade_pair)
+
+    threshold_brake_prices = list(
+        ThresholdBrake.objects.filter(threshold__trade_pair=trade_pair, happened_at__gte=three_second_ago).values_list(
+            'threshold__price', flat=True))
+
+    threshold_brake_prices_str = ', '.join([f'{price}$' for price in threshold_brake_prices])
+
+    last_candle = Candle.objects.filter(trade_pair=trade_pair).order_by('-modified').last()
+    last_candle_high_price = last_candle.high_price if last_candle else None
```

Comment:
It is also very essentially goes to another model. Which and why?

Reply:
This may be a trick question, but in the Candle model we already have a method that returns the last candle. I decided to write another method based on that method, which will return the high price of the last candle